pages/services/spark/2.4.0-2.2.1-3/release-notes/index.md

```diff
@@ -10,20 +10,20 @@ featureMaturity:
 ## Version Spark and Spark History 2.4.0-2.2.1-3
 
 ### New Features
-- Added service name to dispatcher’s VIP endpoints 
-- Added shell-escape fix to spark-cli (SPARK-21014) 
-- Added spark.mesos.executor.gpus (SPARK-21033) 
-- Added dispatcher and driver metrics 
-- Added statsd sink for spark metrics 
+- Added service name to dispatcher’s VIP endpoints
```

Comment:
Not this file but there are two empty files that should be deleted 
`pages/services/spark/2.4.0-2.2.1-3/2.5.0-2.2.1`
`pages/services/spark/2.5.0-2.2.1/2.5.0-2.2.1`